definitions/koa_v2.x.x/flow_>=v0.23.x/koa_v2.x.x.js

```diff
@@ -0,0 +1,163 @@
+type Koa$Request$charset = string;
+type Koa$Request$fresh = bool;
+type Koa$Request$header = Object;
+type Koa$Request$host = string;
+type Koa$Request$hostname = string;
+type Koa$Request$href = string;
+type Koa$Request$idempotent = bool;
+type Koa$Request$ip = string;
+type Koa$Request$ips = string[];
+type Koa$Request$length = ?number;
+type Koa$Request$method = any;
+type Koa$Request$origin = string;
+type Koa$Request$originalUrl = string;
+type Koa$Request$path = string;
+type Koa$Request$protocol = string;
+type Koa$Request$query = Object;
+type Koa$Request$querystring = string;
+type Koa$Request$search = string;
+type Koa$Request$socket = any;
+type Koa$Request$stale = bool;
+type Koa$Request$subdomains = string[];
+type Koa$Request$type = string;
+type Koa$Request$url = string;
+
+declare class Koa$Request extends http$ClientRequest {
+  accepts(...types: string[]): string;
+  acceptsCharsets(...charsets: string[]): string;
+  acceptsEncodings(...types: string[]): string;
+  acceptsLanguages(...charsets: string[]): string;
+  charset: Koa$Request$charset;
+  fresh: Koa$Request$fresh;
+  get(field: string): any;
+  header: Koa$Request$header;
+  headers: Koa$Request$header;
+  host: Koa$Request$host;
+  hostname: Koa$Request$hostname;
+  href: Koa$Request$href;
+  idempotent: Koa$Request$idempotent;
+  ip: Koa$Request$ip;
+  ips: Koa$Request$ips;
+  is(...types: string[]): bool;
+  length: Koa$Request$length;
+  method: Koa$Request$method;
+  origin: Koa$Request$origin;
+  originalUrl: Koa$Request$originalUrl;
+  path: Koa$Request$path;
+  protocol: Koa$Request$protocol;
+  query: Koa$Request$query;
+  querystring: Koa$Request$querystring;
+  search: Koa$Request$search;
```

Comment:
`req: http$IncomingMessage;
  res: http$ClientRequest;`
Should be included here